models/GraphicsContextSettings/GraphicsContextSettings.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+ * the License. A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+/**
+ * Options for exporting an artboard
+ */
+class GraphicsContextSettings {
+  /**
+   *
+   * @param {GraphicsContextSettings.Model} args
+   * @param {GraphicsContextSettings.Model} json
+   */
+  constructor(args = {}, json) {
+    if (json) {
+      Object.assign(this, json);
+    } else {
+      Object.assign(this, GraphicsContextSettings.Model, args);
+    }
+    return this;
+  }
+}
+
+/**
+ * @property {integer} blendMode
+ * @property {float} opacity
+ */
+GraphicsContextSettings.Model = {
```

Comment:
Would like to mention, following discussion in #45 this should be static getter

Reply:
Yeah. I would almost say you merge your PR first and I change my PR.

Reply:
We can do it simultaneously 😉

Reply:
Cherry Pick?

Reply:
no, I mean just change your model

Reply:
@L2jLiga updated according to https://github.com/amzn/sketch-constructor/pull/52